llk_lib/llk_unpack_AB_matmul.h

```diff
@@ -45,15 +45,16 @@ inline void _llk_unpack_AB_matmul_mop_config_(const bool transpose, const std::u
                         TTI_NOP;
                         TTI_NOP;
                         TTI_NOP;
+                        TTI_NOP;
                     } else {
                         TTI_RDCFG(p_gpr_unpack::TMP0, THCON_SEC0_REG3_Base_address_ADDR32);
                         TTI_ADDDMAREG(0, p_gpr_unpack::TMP0, p_gpr_unpack::TMP0, p_gpr_unpack::TILE_SIZE_A);
                         TTI_STALLWAIT(p_stall::STALL_CFG, p_stall::THCON);
                         TTI_WRCFG(p_gpr_unpack::TMP0,0,THCON_SEC0_REG3_Base_address_ADDR32);
                     }
-                    TTI_NOP;
```

Comment:
I think TTI_NOP needs to be added back, at the expense of TTI_NOP at line 48.
TTI_WRCFG instruction takes 2 clocks to propagate the value into the CFG register so you'd need at least 2 instructions between WRCFG and the next instruction that uses the setting WRCFG programs.
In this case, you'll need a NOP and UNPACR_NOP, in most other cases you'd need 2x TTI_NOP.

Reply:
OK, I'll add it back. Didn't know WRCFG takes 2 cycles.
As far as TTI_NOP in line 48 goes, I think it needs to be there so that the number of instructions recorded in replay buffer is the same in case kernel_broadcast = 1 and when it is not.